src/main/java/net/dv8tion/jda/internal/entities/PrivateChannelImpl.java

```diff
@@ -16,56 +16,76 @@
 
 package net.dv8tion.jda.internal.entities;
 
-import net.dv8tion.jda.api.AccountType;
 import net.dv8tion.jda.api.JDA;
 import net.dv8tion.jda.api.entities.ChannelType;
 import net.dv8tion.jda.api.entities.PrivateChannel;
 import net.dv8tion.jda.api.entities.User;
 import net.dv8tion.jda.api.requests.RestAction;
 import net.dv8tion.jda.internal.entities.mixin.channel.middleman.MessageChannelMixin;
+import net.dv8tion.jda.internal.requests.CompletedRestAction;
 import net.dv8tion.jda.internal.requests.RestActionImpl;
 import net.dv8tion.jda.internal.requests.Route;
+import org.jetbrains.annotations.NotNull;
 
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 public class PrivateChannelImpl extends AbstractChannelImpl<PrivateChannelImpl> implements PrivateChannel, MessageChannelMixin<PrivateChannelImpl>
 {
     private User user;
     private long latestMessageId;
 
-    public PrivateChannelImpl(long id, User user)
+    private PrivateChannelImpl(JDA api, long id, User user)
```

Comment:
This constructors are internal. You could just have `public PrivateChannelImpl(JDA api, long id, User user)` and let `user` be `nullable`